src/ocean/text/convert/DateTime_tango.d

```diff
@@ -586,11 +604,9 @@ struct DateTimeLocale
         return f;
     }
 
-    /**********************************************************************
-
-     **********************************************************************/
-
-    private char[] formatCustom (ref Result result, Time dateTime, cstring format)
+    /// Implementation of format
```

Comment:
Would be nice to add doc in the standard format, if you're going to add it. (Despite this module's doc being very non-standard ;)